Packs/MicrosoftWindowsEvents/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,5 @@
+#### Modeling Rules
+
+##### MicrosoftWindowsEvents
+
+- Fixed an issue with the modeling rules.
```

Comment:
@nkanon you can elaborate what we added

Reply:
Understood. I added more info.

Thanks